runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1511,53 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=bad major version %1$u.%2$u of class=%4$.*3$s, the maximum supported major version is %5$u.0; offset=%6$u
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_1=4
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_2=main
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_3=6
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_4=62
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_5=0
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_6=61
```

Comment:
The samples need to be updated to be consistent with the new template (the order has changed):
```
J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_1=62
J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_2=0
J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_3=4
J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_4=main
J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_5=61
J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_6=6
```